### PR TITLE
Release Spark3.5

### PR DESCRIPTION
**Pull request by mahendruajay**
*Issue #, if available:*

*Description of changes:*

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

### CODE REVIEW

File: spark/processing/3.5/py3/docker/py39/Dockerfile.cpu [hunk 1]
```diff
@@ -0,0 +1,142 @@
+FROM public.ecr.aws/amazonlinux/amazonlinux:2023
```
Comment (xiaoxshe) [inline spark/processing/3.5/py3/docker/py39/Dockerfile.cpu:1]: is this the base image recommended? worry is that there is 2023 suffix in the end.

File: spark/processing/3.5/py3/hadoop-config/core-site.xml [hunk 2]
```diff
@@ -0,0 +1,31 @@
...
+         <description>s3a filesystem implementation</description>
+     </property>
+     <property>
+         <name>fs.AbstractFileSystem.s3a.imp</name>
+         <value>org.apache.hadoop.fs.s3a.S3A</value>
+         <description>s3a filesystem implementation</description>
+     </property>
+     <property>
+         <name>fs.s3a.connection.maximum</name>
+         <value>100</value>
```
Comment (xiaoxshe) [inline spark/processing/3.5/py3/hadoop-config/core-site.xml:28]: is this the same configuration with previous release?

File: spark/processing/3.5/py3/docker/py39/Dockerfile.cpu (hunk 1 above)
Comment (mahendruajay) [reply spark/processing/3.5/py3/docker/py39/Dockerfile.cpu:1]: EMR 7.0.0 uses AL2023, we have to use this

File: spark/processing/3.5/py3/hadoop-config/core-site.xml (hunk 2 above)
Comment (mahendruajay) [reply spark/processing/3.5/py3/hadoop-config/core-site.xml:28]: Its the same, all these are copied from previous configs.

File: spark/processing/3.5/py3/nginx-config/nginx.conf [hunk 3]
```diff
@@ -0,0 +1,66 @@
...
+        listen       [::]:80 default_server;
+        server_name  _;
+        root         /usr/share/nginx/html;
+
+        # Load configuration files for the default server block.
+        include /etc/nginx/default.d/*.conf;
+
+        location  /proxy/15050 {
+            proxy_pass http://localhost:15050/;
+        }
```
[the remark this answers is not in the document]
Comment (mahendruajay) [reply spark/processing/3.5/py3/nginx-config/nginx.conf:49]: Its the same, all these are copied from previous configs. You can easily compare with 3.4 dir

**Pull request merged**